DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: hySpc.read.spc
 Type: Package
 Version: 0.0.0.9000
-Title: Read Spectra in .spc Format
-Description: Short Description of the package.
+Title: Import Thermo Galactic's .spc file format into an hyperSpec Object
```

Comment:
@sangttruong: Use title case here, in title.

@bryanhanson Should it be "a hyperSpec object" or "an hyperSpec object"? I think, it should be "a" as it is hollowed by shound "h". What do you think?

Reply:
"a hyperSpec" definitely.

Reply:
@sangttruong, please fix the article too.